README.md

```diff
@@ -7,14 +7,24 @@ for additional information.
 
 ## Process to propose a new lab
 
-Fork the [hyperledger-labs.github.io](https://github.com/hyperledger-labs/hyperledger-labs.github.io) repository.
+1. Fork the [hyperledger-labs.github.io](https://github.com/hyperledger-labs/hyperledger-labs.github.io) repository.
 
-Fill out the [Proposal Template](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/proposal-template.md)
+2. Fill out the [Proposal Template](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/proposal-template.md)
 and save it into the labs subdirectory under the name of your lab,
 such as mynewlab.md. It is expected that your lab repository will have
 the same name so keep that in mind.
 
-Submit a Pull Request.
+Make sure your commit has proper sign-off. This means that your commit

```

Comment:
should this be a step 3. Commit the newly created proposal (even though it is obvious to us)?

Reply:
Well, I see your point but is it really possible someone will want to create a lab and not know that they need to make a commit to make a PR?
I'd say that if that were the case they probably wouldn't know how to make a PR either. Then, where do we stop?

Reply:
Nevermind, I'll make the note a step 3. :-)